src/Dialogs/StripeDialog.vala

```diff
@@ -139,10 +140,14 @@ public class AppCenter.Widgets.StripeDialog : Gtk.Dialog {
 
         get_content_area ().add (layouts);
 
-        var action_area = get_action_area ();
+        var privacy_policy_link = new Gtk.LinkButton.with_label ("https://stripe.com/us/privacy", _("Privacy Policy"));
```

Comment:
Should probably be the region-less https://stripe.com/privacy? I'm not 100% sure.